### PR TITLE
Check that the only children of <dl> are <dt> and <dd>

### PR DESCRIPTION
**Pull request by veerreshr**

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -39,5 +39,13 @@ <h2>Navigation: Only a single navigation element should exist</h2>
     <section>
       <h2>HTML: html tag should have a lang attribute</h2>
     </section>
+    <section>
```
Comment (jackdomleo7) [inline test/index.html:42]: This is my bad for not creating PR templates yet, but could you take a screenshot of this and add it to your PR description? 🙂

File: README.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 - Checks `<li>` is the **only** child of `<ol>` and `<ul>`
 - Checks `<nav>` only exists once on any page
 - Missing `lang` on `<html>`
+- Check that the only children of `<dl>` are `<dt>` and `<dd>`
```
Comment (jackdomleo7) [inline README.md:30]: Nice. Maybe want to shorten this similar to line 27:

```
- Checks `<dt>` and `<dd>` are the **only** children of `<dl>`
```

**Review by jackdomleo7 — CHANGES_REQUESTED** [review]
Nice clean PR! Some comments for you.

Feel free to also add your details to the `contributors` section in `package.json` and ensure you have ran `npm run lint`. 😊

File: test/index.html (hunk 1 above)
Comment (veerreshr) [reply test/index.html:42]: I'm new to contributing, does screenshot mean the picture of this code segment?

File: checka11y.css [hunk 3]
```diff
@@ -65,3 +65,16 @@ nav + nav::after {
   content: "ERROR: <nav> denotes the primary navigation for the page. Other navigations are permitted but should not be wrapped in a <nav>" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
+/**
+ * Description list
```
Comment (jackdomleo7) [inline checka11y.css:70]: I'm happy for this work to go under the "Lists" section 😊

File: test/index.html (hunk 1 above)
Comment (jackdomleo7) [reply test/index.html:42]: No worries 😊 If you open `test.html`, could you take a picture of what you see in this specific section of the webpage? We'd like to see what the work you've done looks like on a web page. https://www.google.com/search?q=how+to+take+a+screenshot

**Comment by jackdomleo7** [comment]
Awesome work @veerreshr!

File: checka11y.css [hunk 4]
```diff
@@ -54,6 +54,15 @@ ul > *:not( li )::after /* All children of <ul> must only be <li> */ {
   font-family: var(--checka11y-font, cursive);
 }
 
+dl > *:not( dt ):not( dd ) {
```
Comment (danspratling) [inline checka11y.css:57]: Can we add in comments to ensure we're super clear why these are here?

```
/* Ensures direct children of <dl> is only <dt> or <dd> */
dl > *:not( dt ):not( dd ) { ... }
dl > *:not( dt ):not( dd )::after { ... }

**Comment by veerreshr** [comment]
> Awesome work @veerreshr!

Thank you so much

File: test/index.html [hunk 5]
```diff
@@ -39,5 +39,13 @@ <h2>Navigation: Only a single navigation element should exist</h2>
...
+      <h2>Description list: Direct children of a &lt;dl&gt; can only be &lt;dt&gt; and &lt;dd&gt;</h2>
+      <dl>
+        <dt>Hello, I'm dt element inside dl element</dt>
+        <dd>Hello, I'm dd element inside dl element</dd>
+        <p>Hello, I'm p(paragraph) element inside dl element. </p>
```
Comment (danspratling) [inline test/index.html:47]: We can probably just use plain text descriptions for these, we don't need to be super strict about stating the tag inside the test of the tag instead preferring readability (so long as it's obvious).

`Hello, I'm a paragraph element inside dl element.`

**Review by danspratling — COMMENTED** [review]
Hey @veerreshr, just a couple of comments here. They're just text changes to make it extra clear what's happening. 

Should be good for approval after they're updated. 👍

**Comment by danspratling** [comment]
Oh Jack merged. @jackdomleo7 These comments are your responsibility now 😉

**Pull request merged**